docs/book/stack-deployment-guide/overview.md

```diff
@@ -0,0 +1,38 @@
+---
+description: An overview of cloud infrastructure for your ML workflows
+---
+
+There can be many motivations behind taking your ML application setup to a cloud environment, from needing specialized compute 💪 for training jobs to having a 24x7 load-balanced deployment of your trained model, serving user requests 🚀. 
+
+With ZenML, there are two kinds of deployments to keep in mind:
+- The ZenML Core infrastructure deployment - this pertains to deploying ZenML itself. Once ZenServer is integrated with the ZenML metadata store, this will hold more relevance so keep an eye out!
+- The ZenML Stack infrastructure deployment - this comprises the resources that form a part of your ZenML stack like orchestrators, artifact stores, model deployers and more!
+
+This guide focuses on the stack infrastructure deployment and has three sections, each corresponding to a common scenario.
+
+## Options for setting up your infrastructure
+
+This section aims to list out different scenarios that you may find yourself in (subject to your background) and helps you to navigate between the available choices.
+
+
+### Manual Deployments 👷
+
+If you're experienced in operations and prefer having the knowledge of every cloud resource that is being set up, intimately, we have a step-by-step guide for AWS, Azure and GCP grouped under [Manual Deployments](./manual-deployments/) with detailed information on every stack component along with other essential things like roles, permissions, connection settings and more. Head over to the page corresponding to your cloud provider to get started!
+
+### One Click Deployments with Stack Recipes ⏩
+
+If you've have never worked on the infrastructure side of things before and don't want to invest my time in it, check out [One Click Deployments with Stack Recipes](./stack-recipes.md) that shows how you can use the ZenML CLI to setup specialized stacks with the execution of a single command 🚀.
```

Comment:
Maybe this could be the first thing you talk about? Seems like it's more the kind of people we want to cater to with these recipes. I.e. it could be the first of the 3 scenarios/options...

Reply:
Great idea! I'll make it the first option in the text and the `toc.md`